hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -527,16 +507,31 @@ protected void writeTableMetadata(HoodieTable table, String instantTime, String
    *
    * @param cleanInstantTime instant time for clean.
    * @param scheduleInline   true if needs to be scheduled inline. false otherwise.
-   * @param skipLocking      if this is triggered by another parent transaction, locking can be skipped.
    */
   @Nullable
+  @Deprecated
```

Comment:
NTR: have deprecated the methods which was taking in "skipLocking" as an argument as we don't need it anymore.